### ISSUE TITLE
Playlist support in cashmusic.js

### ISSUE THREAD

**Issue by jessevondoom** [cashmusic/platform#606]
We already have basic audio player and playlist support in cashmusic.js (https://github.com/cashmusic/cashmusic.js) but we need to add in support for all CASH playlists — including those with secure-file tracks. This means teaching cashmusic.js to unlock tracks on the fly, get a short-expiry link, then stream the new URL. 

Non-secure playlists can easily be supported by simply spitting out the playlist with links and adding the appropriate classes. As close as we can get to that with secured playlists is ideal. My current thinking circles around an encrypted string of the asset ID hashed with the owner's secret. It's not lock-tight. Anyone can call the endpoint and get a URL. Maybe we tie it to the playlist or element so that it could be explicitly turned off by the user. Maybe we only accept POST instead of GET to add a low hurdle to the process. Maybe all of the above and then some. 

Bottom line: we don't need Fort Knox, but I'd like security here to be more difficult than using a double-ended mini-jack cable to create an audio loop-back.
